src/pages/Main/muqtatif/quoteArea/QuoteArea.jsx

```diff
@@ -1,25 +1,60 @@
 import classes from "./QuoteArea.module.scss";
+import { useAtomValue } from "jotai/utils";
+
+import {
+  dropShadowAtom,
+  dropShadowBlurAtom,
+  dropShadowOffsetYAtom,
+  textBgAtom,
+  textBgColorAtom,
+  textBgHeightAtom,
+  textBgOpacityAtom,
+  textBgWidthAtom,
+} from "../muqHeader/settings/window/WindowAtoms";
+import {
+  fontColorAtom,
+  fontFamilyAtom,
+  fontSizeAtom,
+  lineHeightAtom,
+  textAlignAtom,
+} from "../muqHeader/settings/editor/EditorAtoms";
 const QuoteArea = (props) => {
-  const backgroundColor = `rgba(${props.backgroundColor.r}, ${props.backgroundColor.g}, ${props.backgroundColor.b})`;
+  const dropShadow = `drop-Shadow(3px ${useAtomValue(
+    dropShadowOffsetYAtom
+  )}px ${useAtomValue(dropShadowBlurAtom)}px rgba(0, 0, 0, 0.45))`;
+
+  const styles = {
+    width: `${useAtomValue(textBgWidthAtom)}%`,
+    height: `${useAtomValue(textBgHeightAtom)}%`,
+    backgroundColor: useAtomValue(textBgColorAtom),
+    opacity: useAtomValue(textBgOpacityAtom),
+    fontSize: `${useAtomValue(fontSizeAtom)}rem`,
+    color: useAtomValue(fontColorAtom),
+    fontFamily: useAtomValue(fontFamilyAtom),
+    textAlign: useAtomValue(textAlignAtom),
+    lineHeight: useAtomValue(lineHeightAtom),
+    dropShadow: useAtomValue(dropShadowAtom) ? dropShadow : "none",
+  };
```

Comment:
useMemo

Reply:
useAtomValue is a hook  .. so, cant do that